webpack.config.js

```diff
@@ -18,7 +18,11 @@ module.exports = (_env, argv) => {
                 library: 'JitsiMeetJS',
                 libraryTarget: 'umd',
                 path: path.join(process.cwd(), 'dist', 'umd')
-            })
+            }),
+            performance: {
```

Comment:
This is already set on the shared file, any reason to set it here too?

Reply:
Didn't check the purpose of the config from the previous PR. I suppose not, I'll remove it.